apps/coscrad-frontend/src/components/resource-info/resource-info.spec.tsx

```diff
@@ -192,4 +195,35 @@ describe.skip('AllResources', () => {
             await waitFor(() => expect(screen.getByTestId('error')).toBeTruthy());
         });
     });
+
+    it('should apply the custom backgroundColor', () => {
```

Comment:
Isn't this section skipped?  And also wondering why the `<Header>` would be tested here in the resources area?

Reply:
typo, my bad I forgot that it'd need the file name there